Weather App/UI/SettingsVC/SettingsView.swift

```diff
@@ -21,18 +39,165 @@ class SettingsView: UIView {
     }
 
     private func setupView() {
+        setupBackgroundImage()
+        
+        setupCheckButton(celsiusCheckButton, action: #selector(onCelsiusCheckButtonTapped))
+        setupLabel(celsiusLabel, text: "Celsius")
+        setupCheckButton(fahrenheitCheckButton, action: #selector(onFahrenheitCheckButtonTapped))
+        setupLabel(fahrenheitLabel, text: "Fahrenheit")
+        
+        setupImageView(humidityIconImage, image: "humidity")
+        setupCheckButton(humidityCheckButton, action: #selector(onHumidityCheckButtonTapped))
+        setupImageView(pressureIconImage, image: "pressure")
+        setupCheckButton(pressureCheckButton, action: #selector(onPressureCheckButtonTapped))
+        setupImageView(windIconImage, image: "wind")
+        setupCheckButton(windCheckButton, action: #selector(onWindCheckButtonTapped))
+    }
+    
+    private func setupBackgroundImage() {
         backgroundImage.translatesAutoresizingMaskIntoConstraints = false
         backgroundImage.image = UIImage(named: "background")
         backgroundImage.contentMode = .scaleAspectFill
         self.addSubview(backgroundImage)
     }
     
+    private func setupCheckButton(_ checkButton: UIButton, action: Selector) {
+        checkButton.translatesAutoresizingMaskIntoConstraints = false
+        checkButton.layer.backgroundColor = UIColor.white.cgColor
+        checkButton.addTarget(self, action: action, for: .touchUpInside)
+        self.addSubview(checkButton)
+    }
+    
+    private func setupLabel(_ label: UILabel, text: String) {
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = text
+        label.font = UIFont.systemFont(ofSize: 30)
+        self.addSubview(label)
+    }
+    
+    private func setupImageView(_ imageView: UIImageView, image: String) {
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = UIImage(named: image)
+        self.addSubview(imageView)
+    }
+    
     private func setupConstraints() {
+        let SPACE_BETWEEN_ELEMENTS: CGFloat = 20
```

Comment:
Where is this comming from? :D

Reply:
Manually defined constant that sets up the view the best (in iPhone 12 Pro Max). Should be changed later when I start testing the app for other device sizes.

Reply:
But when you define properties, you should use camel case, not this "style".